source/features/lines-changed-per-commit.tsx

```diff
@@ -0,0 +1,52 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+
+const getCommitChanges = async (commit: string): Promise<number[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+            object(expression: "${commit}") {
+                ... on Commit {
+                    additions
+                    deletions
+                    }
+                }
+		}
+    `);
+	// @ts-ignore
+	const {additions, deletions} = repository.object;
+	return [additions, deletions];
+};
+
+async function init(): Promise<void | false> {
+	const pageCommit = (await elementReady('.sha.user-select-contain'))!;
```

Comment:
```suggestion
	const commitSha = (await elementReady('.sha.user-select-contain'))!.textContent!;`
```

You can merge these 2 lines.